### DISCUSSION TITLE
After updating the compose file, and compose down, compose up, can't find my old data.

### DISCUSSION THREAD

**Discussion by baleine702** [immich-app/immich#13526] [Q&A]
I've used immich with unraid.

Couldn't start the stack after update.

Followed one of the discussion to update the compose file then compose down and up. (https://github.com/immich-app/immich/discussions/12647)

I now have a fresh install.

How can I get my files back?

Thank you very much.

**Comment by bo0tzz** [comment]
You need to post more detail than this. What was your compose file before, what changes did you make, what's your .env, etc?

**Comment by baleine702** [comment]
My current compose here.
[docker-compose.txt](https://github.com/user-attachments/files/17410989/docker-compose.txt)

I haven't saved the previous one. It was an old one with micro-services.

Current Env File :
[env.txt](https://github.com/user-attachments/files/17411011/env.txt)

Previous one was the same without  the " DB_DATA_LOCATION=/var/lib/postgresql/data" part.

All pictures seems to still be in the upload location when I look on the filesystem.

**Reply by bo0tzz** [reply]
> Previous one was the same without the " DB_DATA_LOCATION=/var/lib/postgresql/data" part.

You probably blindly applied the changes from https://github.com/immich-app/immich/releases/tag/v1.102.0

**Reply by baleine702** [reply]
I just read that now.
What I've done is, blindly follow update guide from unraid install page.
When it broke and had same issue as https://github.com/immich-app/immich/discussions/12647, I then followed advice to update compose file.

So here I am now. Can I go back? Or is it lost as stated in the link you gave me?

**Reply by bo0tzz** [reply]
You can change back the volume mount, and if you're lucky it'll pick up the old database and work.
